YunoHost/yunohost
# [enh] Bootprompt is back

## Problem
User are a bit lost after installation of yunohost since the boot prompt has disappear in YunoHost 2.4.

## Suggested solution
Display the boot prompt has before
![yunoprompt](https://user-images.githubusercontent.com/4080016/29951782-5c4ca1dc-8ec5-11e7-8279-e0e61a63f109.png)
Note: I have remove uptime and Main domain

## How to test
To test you need to put 
bin/yunoprompt in /usr/bin/
data/other/yunoprompt.service in /etc/systemd/system/

chmod u+x /usr/bin/yunoprompt


After that run systemctl enable yunoprompt and restart a VM with a VNC console. You will see the new boot prompt.

## PR Status
Opinion needed

## Validation

- [x] **Principle agreement** 2/2 : Aleks, opi
- [x] **Quick review** 1/1 : opi
- [x] **Deep review**1/1: Aleks
